README.md

```diff
@@ -107,11 +107,15 @@ After you clone the repository locally, there are several configuration steps re
 
 ##### CMS
 
-- Copy `server/conf/cms/secrets.sample.py` to `server/conf/cms/secrets.py`
+Add CMS settings and secrets:
 
-- To emulate a specific CMS project, copy https://github.com/TACC/Core-CMS-Resources/blob/main/__PROJECT_TO_EMULATE__/settings_custom.py to `server/conf/cms/settings_custom.py`
+```sh
+curl "https://cdn.jsdelivr.net/gh/TACC/Core-CMS@main/taccsite_cms/secrets.example.py" -o server/conf/cms/settings_custom.py
+cp server/conf/cms/secrets.sample.py server/conf/cms/secrets.py
+curl "https://cdn.jsdelivr.net/gh/TACC/Core-CMS@main/taccsite_cms/settings_local.example.py" -o server/conf/cms/settings_local.py
+```
```

Comment:
**Opting for ready-to-go CMS settings instead.** This was a means to prevent directories from being created if files have not. Short-term fix is instead to tell any dev _that wants to customize the CMS (which is optional)_ to delete the directories and create the files and restart.